README.md

```diff
@@ -0,0 +1,202 @@
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magician <a name="about-project"></a>
+<!-- #Project presentation(https://www.loom.com/share/f39c71dbc7814c3283287712da5e8f9e) -->
+
+ This is a simple react calculator.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project was built using the following stack:
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>JavaScript</li>
+    <li>CSS</li>
+    <li>Markdown</li>
+    <li>Gitflow</li>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>No server side tech used</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>No Database used</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Feature <a name="key-features"></a>
+
+- [x] React setup
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo 
+
+N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+In order to run this project you need have node installed
+[Node.js](https://nodejs.org/en)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Paulinelali/math-magicians/
+```
+
+### Usage
+
+To run the project, do the following:
+
+Open the folder where you've cloned the project
+in your terminal run npm start
+
+
+### Run tests
+
+no command line test is require for this project
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+using netlify
+github pages
+your prefer platform
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ayomide Sunday Emmanuel**
+
+- [GitHub:](https://github.com/Paulinelali)
+- [Linkedin:](https://www.linkedin.com/in/ayomide-emmanuel-1353b31b9/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+I will be doing but not limited to the following:
+- [x] calculator functionality
+- [x] > Add user interaction
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/) or contact me direct(https://www.linkedin.com/in/ayomide-emmanuel-1353b31b9/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project...
+Give it a start and contribute
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] @Paulinelali Kindly consider adding an MIT license to your project, it's currently missing making the MIT link to rent the error below 
![image](https://github.com/Paulinelali/math-magicians/assets/61408860/a26de870-2f5b-4d72-b97c-7b33dd10f100)

Reply:
Hi, @danmainah Please can you go the extra mile by telling me specifically boiler templates to delete? I'm kind of not really sure what to do here.

Thank you so much for your help